mission_control/navigator_missions/nav_missions/start_gate.py

```diff
@@ -168,16 +183,10 @@ def get_size_and_build_walls(self):
         b_rot_mat = trns.euler_matrix(0, 0, b_theta)[:3, :3]
 
         # Make the endpoints
-        rot_buffer = b_rot_mat.dot([self.buffer, 0, 0])
+        rot_buffer = b_rot_mat.dot([20, 0, 0])
```

Comment:
The new method I'm using doesn't need the buffer at all in reality but I still want to expand the grid more than it has to be